unity-client/ProjectSettings/QualitySettings.asset

```diff
@@ -21,7 +21,7 @@ QualitySettings:
     skinWeights: 4
     textureQuality: 0
     anisotropicTextures: 2
-    antiAliasing: 2
+    antiAliasing: 0
```

Comment:
Was this change voluntary? If not, you may want to revert it :)

Reply:
@pablitar No, It was generated when I ran unity, If it isn't necessary, I'll revert it